### ISSUE TITLE
Add installation info in README.md

### ISSUE THREAD

**Issue by emigenix** [rprichard/winpty#91]
Please add instalaltin info in the README.md to explain how to install the files as given under `releases` wiki.

**Comment by mintty** [comment]
`/usr/local` is not used by cygwin packages, and does not even exist in a typical cygwin installation.
Installation instructions should target `/usr/bin`.

**Comment by emigenix** [comment]
> /usr/local is not used by cygwin packages,

@mintty Exactly. That makes sense for stuff that can be installed with the package manager, everything else should be kept isolated from default location. At least until packages can be easily installed,updated or removed with a manager app.

**Comment by javacom** [comment]
To install in cygwin x64, type these commands
`wget --no-check-certificate https://github.com/rprichard/winpty/releases/download/0.4.2/winpty-0.4.2-cygwin-2.6.1-x64.tar.gz`
`mkdir -p /usr/local`
`tar xzvf winpty-0.4.2-cygwin-2.6.1-x64.tar.gz -C /usr/local/ --strip 1`

**Comment by rprichard** [comment]
Why should certificate checking be disabled?

**Comment by javacom** [comment]
> Why should certificate checking be disabled?

because user did not install the cygwin package **ca-certificates** by default
and have to specify the --ca-directory=/usr/ssl/certs
